# request
string input_msg

---
# response 
string output_msg 